lmms_eval/models/llava_vid.py

```diff
@@ -59,6 +59,8 @@ def __init__(
         mm_spatial_pool_mode: str = "average",
         overwrite: bool = True,
         video_decode_backend: str = "pyav",
+        delay_load: bool = False,
+        tie_weights: bool = True,
```

Comment:
Same as above, should this default to True?

Reply:
Same thing as above. We aren't doing video anyways, so let's keep the change.